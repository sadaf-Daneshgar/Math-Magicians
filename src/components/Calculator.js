import React from 'react';
import './style.css';
import InputNumber from './Button';

function Calculator() {
  return (
    <>
      <div className="container">
        <InputNumber className="numInput" value="0" />
        <div className="rows-contanier">
          <div className="horizontal-rows">
            <button className="num-buttons" type="button">
              AC
            </button>
            <button className="num-buttons" type="button">
              +/-
            </button>
            <button className="num-buttons" type="button">
              %
            </button>
            <button className="sign-btn" type="button">
              +
            </button>
          </div>
          <div className="horizontal-rows">
            <button className="num-buttons" type="button">
              7
            </button>
            <button className="num-buttons" type="button">
              8
            </button>
            <button className="num-buttons" type="button">
              9
            </button>
            <button className="sign-btn" type="button">
              *
            </button>
          </div>
          <div className="horizontal-rows">
            <button className="num-buttons" type="button">
              4
            </button>
            <button className="num-buttons" type="button">
              5
            </button>
            <button className="num-buttons" type="button">
              6
            </button>
            <button className="sign-btn" type="button">
              -
            </button>
          </div>
          <div className="horizontal-rows">
            <button className="num-buttons" type="button">
              1
            </button>
            <button className="num-buttons" type="button">
              2
            </button>
            <button className="num-buttons" type="button">
              3
            </button>
            <button className="sign-btn" type="button">
              +
            </button>
          </div>
          <div className="horizontal-rows">
            <button className="num-zero" type="button">
              0
            </button>
            <button className="num-buttons" type="button">
              .
            </button>
            <button className="sign-btn" type="button">
              =
            </button>
          </div>
        </div>
      </div>
    </>
  );
}

export default Calculator;
