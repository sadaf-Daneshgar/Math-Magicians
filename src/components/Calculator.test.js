import React from 'react';
import renderer from 'react-test-renderer';
import Calculator from './Calculator';

test('Calculator component snapshot', () => {
  const tree = renderer.create(<Calculator />).toJSON();
  expect(tree).toMatchSnapshot();
});
