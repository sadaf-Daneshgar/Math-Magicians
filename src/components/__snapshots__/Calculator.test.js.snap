// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Calculator component snapshot 1`] = `
Array [
  <p
    className="math-title"
  >
    Let\`s do some math!
  </p>,
  <div
    className="container"
  >
    <div
      className="numInput"
    >
      0
    </div>
    <div
      className="rows-contanier"
    >
      <div
        className="horizontal-rows"
      >
        <button
          className="num-buttons"
          onClick={[Function]}
          type="button"
        >
          AC
        </button>
        <button
          className="num-buttons"
          onClick={[Function]}
          type="button"
        >
          +/-
        </button>
        <button
          className="num-buttons"
          onClick={[Function]}
          type="button"
        >
          %
        </button>
        <button
          className="sign-btn"
          onClick={[Function]}
          type="button"
        >
          +
        </button>
      </div>
      <div
        className="horizontal-rows"
      >
        <button
          className="num-buttons"
          onClick={[Function]}
          type="button"
        >
          7
        </button>
        <button
          className="num-buttons"
          onClick={[Function]}
          type="button"
        >
          8
        </button>
        <button
          className="num-buttons"
          onClick={[Function]}
          type="button"
        >
          9
        </button>
        <button
          className="sign-btn"
          onClick={[Function]}
          type="button"
        >
          x
        </button>
      </div>
      <div
        className="horizontal-rows"
      >
        <button
          className="num-buttons"
          onClick={[Function]}
          type="button"
        >
          4
        </button>
        <button
          className="num-buttons"
          onClick={[Function]}
          type="button"
        >
          5
        </button>
        <button
          className="num-buttons"
          onClick={[Function]}
          type="button"
        >
          6
        </button>
        <button
          className="sign-btn"
          onClick={[Function]}
          type="button"
        >
          -
        </button>
      </div>
      <div
        className="horizontal-rows"
      >
        <button
          className="num-buttons"
          onClick={[Function]}
          type="button"
        >
          1
        </button>
        <button
          className="num-buttons"
          onClick={[Function]}
          type="button"
        >
          2
        </button>
        <button
          className="num-buttons"
          onClick={[Function]}
          type="button"
        >
          3
        </button>
        <button
          className="sign-btn"
          onClick={[Function]}
          type="button"
        >
          +
        </button>
      </div>
      <div
        className="horizontal-rows"
      >
        <button
          className="num-zero"
          onClick={[Function]}
          type="button"
        >
          0
        </button>
        <button
          className="num-buttons"
          onClick={[Function]}
          type="button"
        >
          .
        </button>
        <button
          className="sign-btn"
          onClick={[Function]}
          type="button"
        >
          =
        </button>
      </div>
    </div>
  </div>,
]
`;
